tests/unit/amo/components/TestPermissionsCard.js

```diff
@@ -21,13 +21,14 @@ describe(__filename, () => {
     store = dispatchClientMetadata().store;
   });
 
-  const createVersionWithPermissions = (permissions) => {
+  const createVersionWithPermissions = (optional, required) => {
```

Comment:
We probably want to switch to a unique object argument:

```suggestion
  const createVersionWithPermissions = ({ optional = [], required = [] }) => {
```

it will be easier to read in the different test cases.

Reply:
I was thinking about that when going through the test cases. Good suggestion.